tests/run/typeclass-derivation2b.scala

```diff
@@ -49,7 +49,7 @@ object Lst {
       case x: Cons[_] => 0
       case Nil => 1
     }
-    inline override def alternative(inline n: Int) <: GenericProduct[_ <: Lst[T]] =
+    inline override def alternative(n: Int) <: GenericProduct[_ <: Lst[T]] =
```

Comment:
For this one, you should instead make the parameter of the abstract inline def being implemented an inline parameter.

Reply:
Oh, the override keyword made me think it was an override. Will change it.